src/dug/_version.py

```diff
@@ -1 +1 @@
-__version__ = "2.6.dev0"
+__version__ = "2.6.dev1"
```

Comment:
Since we don't really release develop builds into the wild for this project, you don't have to worry about updating this. We may just want to set this to not have a trailing build number to better communicate that workflow